real_time/arduino_src/radio_buggy_mega/main.cpp

```diff
@@ -581,12 +603,12 @@ int main(void)
             g_errors &= ~_BV(RBSM_EID_AUTON_LOST_SIGNAL);
         }
 
-        // For the old buggy, the voltage divider is 10k ohm on the adc side and
-        // 16k ohm on top.
-        // Calculated map normally set to 13000, but the avcc is 4.86 volts
-        // rather than 5.
-        g_current_voltage = adc_read_blocking(BATTERY_ADC);
-        g_current_voltage = map_signal(g_current_voltage, 0, 255, 0, 12636); // in millivolts
+        // Voltage divider mounted with 24k/12k ohm resistors
+        // Constants obtained through caracterisation of divider and ADC reader
+        // scanning the full range and then doing a linear regression
```

Comment:
Please move these up where the constants are #defined.  Also the wording is a little awkward